src/org/traccar/processing/ComputedAttributesHandler.java

```diff
@@ -36,10 +39,32 @@ public ComputedAttributesHandler() {
         engine.setStrict(true);
     }
 
+    private MapContext prepareContext(Position position) {
+        MapContext result = new MapContext();
+        Method[] methods = position.getClass().getMethods();
+        for (Method method : methods) {
+            if (method.getName().startsWith("get") && method.getParameterTypes().length == 0
+                    && !method.getName().equals("getClass")) {
```

Comment:
I suggest some more generic solution for excluding methods here. Something like this:
```java
Set<Method> methods = new HashSet<>(Arrays.asList(position.getClass().getMethods()));
methods.removeAll(Arrays.asList(Object.class.getMethods()));
for (Method method : methods) {
    ...
```